plugin.rb

```diff
@@ -34,33 +34,33 @@ def after_authenticate(auth_token)
   end
 
   def register_middleware(omniauth)
-    omniauth.provider :microsoft_office365,
+    omniauth.provider :microsoft_365,
```

Comment:
To change this, you also need to rename the Omniauth::Strategies::MicrosoftOffice365 class